samples/HostedInAspNet.Client/HostedInAspNet.Client.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Razor/2.1.0-preview2-30159">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Exe</OutputType>
```

Comment:
/cc @SteveSandersonMS  currently the 'build' command expects that this project will be an exe (`GetEntryPoint`) not a library. WebSDK forces all projects to be an exe, which is the reason for the delta.

Any thoughts on this? bug or intentional? I can investigate if needed

Reply:
Correct, the assembly does need to indicate its entrypoint.

> Any thoughts on this? bug or intentional? I can investigate if needed

Sorry, I don't follow - what might be a bug? The absence of "<OutputType>exe</OutputType>" before? It was only omitted because it was redundant (like you say, the Web SDK put it there by default).

If we're changing to the Razor SDK - which I agree we should do for now until we have a Blazor SDK that extends the Razor one - then it looks totally reasonable to put `<OutputType>Exe</OutputType>` back.

Or am I missing the question?

Reply:
No, I think you we're talking about the same thing. Makes sense to me now.